tests/queue/queue_shortcuts_explicit_fp16.cpp

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2022 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "../../util/extensions.h"
+#include "../common/common.h"
+#include "../common/disabled_for_test_case.h"
+#include "queue_shortcuts_explicit.h"
+
+namespace queue_shortcuts_explicit_fp16 {
+
+using namespace sycl_cts;
+using namespace queue_shortcuts_explict;
+
+// hipSYCL, DPCPP not tested
```

Comment:
We have both of them in CI, so I suggest enable and check the status in the logs.

Reply:
I have removed the checks for hipSYCL, but the queue tests (already) are in the CI filter file for hipSYCL, so the test will not be executed.

I have made the comment for DPCPP more specific, the check is in place due to the test not compiling.